ui/src/components/OverviewTable.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Table, TableHeader, TableBody, sortable, SortByDirection } from '@patternfly/react-table';
+import { Redirect } from 'react-router-dom';
+
+class OverviewTable extends React.Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      columns: [
+        { title: 'Apps', transforms: [ sortable ] },
+        { title: 'Num of Deployed Versions', transforms: [ sortable ] },
+        { title: 'Num of Clients', transforms: [ sortable ] },
+        { title: 'Numb of App Sartups', transforms: [ sortable ] }
+      ],
+      rows: [
+        [ 'App-A', 3, 245, 4 ],
+        [ 'App-B', 4, 655, 5 ],
+        [ 'App-C', 1, 970, 6 ],
+        [ 'App-D', 6, 255, 7 ],
+        [ 'App-E', 5, 120, 8 ]
+      ],
+
+      sortBy: {},
+      redirect: false
+    };
+    this.onSort = this.onSort.bind(this);
+    this.onRowClick = this.onRowClick.bind(this);
+  }
+
+  onRowClick (event, rowId, props) {
+    console.log('navigate was called on row: ', rowId.apps.title);
+    this.setState({ redirect: true });
```

Comment:
Is this the only method of redirecting to another page?

Reply:
I think there are better ways. This was only to show that the row is clickable and we can respond to it, which I wasn't sure was available in the older implementation. I can remove this for now until we revisit it.

Reply:
I am not familar with the react router, I wonder if there is a way through that to redirect without using this method.

Reply:
@davidffrench it using the react router here: https://github.com/aerogear/mobile-security-service/pull/53/files#diff-02e05267c5acd6382ef9f4a9ed5a3092R49 i'm just using a boolean to check if it should redirect on rerender. This definitely can be improved but I don't want to make the scope of this pr too big. I would prefer to make a follow on JIRA to improve this linking. It was more of a poc around the row click for table in PF4 than a proper implementation.

Reply:
That is perfect, a new JIRA sounds good to me.